test/functional/scan/test-scan-inclusive-tbb.cpp

```diff
@@ -0,0 +1,22 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "test-scan-inclusive.hpp"
+
+#if defined(RAJA_ENABLE_TBB)
+
+// TBB policy types to test
+using TBBExecTypes = list< RAJA::tbb_for_exec >;
```

Comment:
Same here tbb_for_static, tbb_for_dynamic

Reply:
Got it. Thanks.